mastermind.rb

```diff
@@ -55,21 +82,42 @@ def clean_guess(guess)
     rescue NoMethodError, TypeError
       puts "Please enter only letters."
     end
-    return guess
+    guess
   end
 
   def validate_length(guess)
     if guess.length != 4
-      puts "Please enter only four colors."
+      valid_length = false
+      puts "Please enter four colors using only their first letters."
```

Comment:
Actually, I'd probably remove the puts line and just have this be `guess.length != 4` and then have the error be in the else clause of  `input_guess`

Reply:
Great idea - thank you!